.github/workflows/create-new-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: new PR greeter
+on:
+  pull_request_target:
+    types: [opened,reopened]
+
+jobs:
+  new_pr_greeter:
+    if: github.event.issue.pull_request &&
+      startsWith(github.event.comment.body, '/test-osp-extended') && (
+      github.event.comment.author_association == 'MEMBER' ||
+      github.event.comment.author_association == 'COLLABORATOR')
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout forklift
+        uses: actions/checkout@v3
```

Comment:
do we really need to checkout the forklift repo here?

Reply:
apparently not, fixed.